tools/rapids-get-pr-conda-artifact-github

```diff
@@ -23,20 +23,23 @@ commit="${4:-}"
 
 # If commit is not provided, get the latest commit on the PR
 if [[ -z "${commit}" ]]; then
-    commit=$(rapids-retry gh pr view "${pr}" --repo rapidsai/"${repo}" --json headRefOid --jq '.headRefOid')
+    commit=$(git ls-remote https://github.com/rapidsai/"${repo}".git refs/heads/pull-request/"${pr}" | cut -f1)
 fi
 
 # Use a subshell to isolate environment variable changes
 (
     # Set up environment variables needed for GitHub artifact download
-    export RAPIDS_REPOSITORY="rapidsai/${repo}"
-    export RAPIDS_BUILD_TYPE="pull-request"
-    export RAPIDS_SHA="${commit}"
-    export RAPIDS_REF_NAME="pull-request/${pr}"
+    RAPIDS_REPOSITORY="rapidsai/${repo}"
+    RAPIDS_SHA="${commit}"
+    RAPIDS_REF_NAME="pull-request/${pr}"
 
     # Generate the artifact name
-    artifact_name="$(RAPIDS_NO_PKG_EXTENSION=true rapids-package-name "conda_${package_type}")"
+    pkg_name="$(RAPIDS_NO_PKG_EXTENSION=true rapids-package-name "conda_${package_type}")"
+    github_run_id=$(gh run list --repo "${RAPIDS_REPOSITORY}" --branch "${RAPIDS_REF_NAME}" --commit "${RAPIDS_SHA}" --json databaseId --jq '.[0] | .databaseId')
```

Comment:
Does this need to use `rapids-retry` in case the GitHub API fails? Will the `$(...)` parse the output correctly if it is retried?
```suggestion
    github_run_id=$(rapids-retry gh run list --repo "${RAPIDS_REPOSITORY}" --branch "${RAPIDS_REF_NAME}" --commit "${RAPIDS_SHA}" --json databaseId --jq '.[0] | .databaseId')
```